x-pack/solutions/security/packages/features/src/security/v2_features/kibana_sub_features.ts

```diff
@@ -728,6 +781,16 @@ export const getSecurityV2SubFeaturesMap = ({
       SecuritySubFeatureId.endpointExceptions,
       enableSpaceAwarenessIfNeeded(endpointExceptionsSubFeature()),
     ],
+
+    ...((experimentalFeatures.endpointManagementSpaceAwarenessEnabled
+      ? [
+          [
+            SecuritySubFeatureId.globalArtifactManagement,
+            enableSpaceAwarenessIfNeeded(globalArtifactManagementSubFeature()),
```

Comment:
NIT: It does not seem necessary to call `enableSpaceAwarenessIfNeeded` for this subFeature, it does not change anything. But it's okay for consistency.